flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/common/CommonExecTableSourceScan.java

```diff
@@ -69,14 +69,19 @@ protected CommonExecTableSourceScan(
         this.tableSourceSpec = tableSourceSpec;
     }
 
+    @Override
+    public String getSimplifiedName() {
+        return tableSourceSpec.getObjectIdentifier().getObjectName();
```

Comment:
use full identifier to avoid name conflicts ?

Reply:
I think it is enough in most of the case, it is rare that we are reading from different sources with the same name in a job, and if there is conflict, we can check details in description